uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/dnd/GridWidgetDnDMouseMoveHandler.java

```diff
@@ -596,7 +597,9 @@ protected void handleRowMove(final NodeMouseMoveEvent event) {
         final List<GridColumn<?>> allGridColumns = activeGridModel.getColumns();
 
         final BaseGridRendererHelper rendererHelper = activeGridWidget.getRendererHelper();
+        final BaseGridRendererHelper.RenderingInformation renderingInformation = rendererHelper.getRenderingInformation();
```

Comment:
Mmmm ok so here it seems that `rendererHelper.getRenderingInformation()` is reliable. I'm a bit confused

Reply:
I changed this to get all rows heights once and re-use the collection in the remainder of the method. However on review getting ALL rows' heights (all be it once) is probably slower than getting the height of the rows involved in the drag operation (which, since is only visible rows, is less for large tables). I have therefore reverted the change to this method.